desktop-src/Services/protecting-anti-malware-services-.md

```diff
@@ -151,6 +151,9 @@ Note that the certificate chain must include the Code Signing EKU (1.3.6.1.5.5.7
 > [!Note]  
 > If EKU information is included in certificate information for the ELAM driver, then the same EKU must be used when signing your binaries.
 
+
+> [!Note]
+> Windows code integrity has limitation to string representation of OID in EKU up to 64 characters (including zero tetmination character).
```

Comment:
Looks like a typo, `tetmination` instead of `termination`.

Reply:
> Looks like a typo, `tetmination` instead of `termination`.

You are correct. I've pushed fix.

Reply:
I find useful document this limitation as we faced such issue (on single EKU).

I’m not sure if this limitation applies on
a. single EKU, in semicolon delimited EKUs resource string, or
b. on whole EKU line entry

I hope Microsoft documentation team will verify this with Windows Code Integrity team to make this documentation precise.